src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -19,23 +19,23 @@ public abstract class ApplicationDeployer : IApplicationDeployer
     {
         private readonly Stopwatch _stopwatch = new Stopwatch();
 
+        protected readonly string DotnetCommandName = "dotnet";
```

Comment:
why are these `protected readonly`? Can't they be just `public static` (possibly on some const class?) so that they can be used in other places?